dali/operators/reader/loader/webdataset/tar_utils.h

```diff
@@ -61,6 +45,22 @@ class DLL_PUBLIC TarArchive {
    * @brief Returns whether it has reached the end of archive.
    */
   bool EndOfArchive() const;
+  /**
+   * @brief Sets the offset to which the stream pointer should go.
+   * @remark The offset must point to a file header; other values will cause undefined behaviour.
+   */
+  void Seek(size_t offset);
+
+  enum EntryType
+  {
+    ENTRY_FILE = 0,
```

Comment:
I think that maybe there should be some kind of null entry?
```suggestion
    ENTRY_NONE = 0,
    ENTRY_FILE,
```

Reply:
done